xrdp/xrdp_types.h

```diff
@@ -382,6 +382,9 @@ struct xrdp_wm
   int hints;
   char pamerrortxt[256];
 
+  /* Disable checks for channel input when channels are unavailable */
+  int disable_channel_input_check;
```

Comment:
First of all, I'm not strong position on this. However I concern about this may cause double-negative expression and spoil code-readability. If you have a fair reason to choose `disabled_`, it's OK.